examples/screen_cast_pw.rs

```diff
@@ -0,0 +1,218 @@
+use std::os::fd::{IntoRawFd, OwnedFd};
+
+use ashpd::desktop::{
+    screencast::{CursorMode, Screencast, SourceType, Stream as AshStream},
+    PersistMode,
+};
+use pipewire as pw;
```

Comment:
Maybe rename the crate in Cargo.toml, using `pw = {package = "pipewire"...}`?

Reply:
well we could but our users might get confused the moment they try to copy & paste the source directly in their project? their setup will by default not resolve it correctly?

Reply:
well, that is true :) but it is something i expect a rust user to be familiar with imho

Reply:
haha alright sure

Reply:
looks like the `pw::spa::pod::object!` macro from pipewire expects the module `pipewire` to exist

```rust
#[doc(hidden)]
#[macro_export]
macro_rules! __object__ {
    ($type_:expr, $id:expr, $($properties:expr),* $(,)?) => {
        pipewire::spa::pod::Object { // <-- here
            type_: $type_.as_raw(),
            id: $id.as_raw(),
            properties: [ $( $properties, )* ].to_vec(),
        }
    };
}
```

and.. a fix is to do the opposite 🙃

```
use pw as pipewire
```

they should've used $crate instead lol

----------------------------

now should we umm report this to pipewire folks or just leave pipewire as is?

Reply:
it would be nice to report it nonetheless but for now i think just using pipewire as pw is ok :)

Reply:
cool! I'll report it on the side